src/adagrams.js

```diff
@@ -1,8 +1,116 @@
-const Adagrams = {
+class Adagrams{
+
+
   drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+
+    const letters = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E","E", "E","E", "E" ,"E", "E","E", "E", "E", "E", 
+    "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", 
+    "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S",
+    "T", "T", "T", "T",  "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"];
+    let hand = new Array(10);
+    //found the random function explaination on stack overflow 
+    for (let i = 0; i < hand.length; i++){
+      let rand = Math.floor(Math.random() * Math.floor(letters.length));
+      letters.splice(rand, 1);
+      hand.splice(i, 1, letters[rand]);
+    }
+    return hand; 
+  }
+
+  usesAvailableLetters(input, lettersInHand) {
```

Comment:
While this function passes the provided tests, it doesn't return the correct boolean in all cases. For instances ` `ABC`.includes(`AC`)`  returns `false`.